grabthar.yml

```diff
@@ -2,6 +2,9 @@ entrypoint: ./main.ts
 exclude:
   - bin
   - packages
+target: es2015
```

Comment:
This target is necessary for `QJSEngine`. I should check the other packages and ensure they're building to a similar target.